source/manuals/accessibility.html.md.erb

```diff
@@ -79,12 +79,14 @@ You can test your service using an [accessibility empathy lab](https://gds.blog.
 
 Testing with real users can be a time consuming process. It is recommended that this is done only after other accessibility testing takes place.
 
-### Consider using the GOV.UK Design System
+### Use the GOV.UK Design System
 
 The [GOV.UK Design System](https://design-system.service.gov.uk/) is a collection of website styles, components and patterns designed to be used to build government services. It provides pre-built website markup that has been developed specifically to be accessible. Using it will also save development time and provide a look and feel consistent with other government services.
 
 While it may not have a full set of components for every service, it provides a solid foundation to work from that can be expanded to suit a service's individual needs.
 
+The GOV.UK Design System also includes the [GOV.UK Prototype Kit](https://govuk-prototype-kit.herokuapp.com/), which can be used to show ideas to people you work with and to do user research.
```

Comment:
nice updates! I think I'd drop the 'includes' part, something like

You can also use the [GOV.UK Prototype Kit](https://govuk-prototype-kit.herokuapp.com/), to quickly create working prototypes to share ideas with people and to do user research.

Reply:
Thanks @joelanman - that seems sensible to me 👍
I've dropped the 'includes' part and tightened up the content a bit. See if this looks good to you:

> You can also use the [GOV.UK Prototype Kit](https://govuk-prototype-kit.herokuapp.com/) to quickly create working prototypes, share ideas with people and conduct user research.